File: tools/echo.py
BOLD = '\033[1m'
RESET = '\033[0m'
LEVELS = {
    "OK": { "color": '\033[32m', "level": "Ok" },
    "INFO": { "color": '\033[33m', "level": "Info" },
    "ERROR": { "color": '\033[31m', "level": "Error" },
}

def __color(level):
    return LEVELS[level]['color']

def __level(level):
    return LEVELS[level]['level']

def echo(message, level = "INFO"):
    print(f"{__color(level)}{BOLD}[ {__level(level)} ]:{RESET} {message}")

File: scripts/utilities/echo.mjs
const info = (msg) => {
  const header = chalk.blue('[ Info ]');
  echo(header, msg);
};

const error = (msg) => {
  const header = chalk.red('[ Error ]');
  echo(header, msg);
};

const success = (msg) => {
  const header = chalk.green('[ Success ]');
  echo(header, msg);
};

const echo = (header, msg) => {
  console.log(`${chalk.bold(header)} ${msg}`);
};

export { info, error, success };
